integration_test/third_party_apps_data/applications/tomcat/metadata.yaml

```diff
@@ -107,3 +107,54 @@ expected_logs:
   - name: jsonPayload.user
     type: string
     description: Authenticated username for the request
+configuration_options:
+  logs:
+  - type: tomcat_system
```

Comment:
![image](https://user-images.githubusercontent.com/8354694/164293598-6de1b102-9692-4850-b604-a87df36a17ed.png)
There is some unmarshall error

Reply:
Fixed.